src/components/BasicSection/BasicSection.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import './BasicSection.css';
+
+function BasicSection(props) {
```

Comment:
This is only used in 1 place, and is a tiny file. No need to have a separate component.